pinot-common/src/main/java/org/apache/pinot/common/metadata/ZKMetadataProvider.java

```diff
@@ -112,14 +112,14 @@ public static String constructPropertyStorePathForInstancePartitions(String inst
     return StringUtil.join("/", PROPERTYSTORE_INSTANCE_PARTITIONS_PREFIX, instancePartitionsName);
   }
 
-  public static String constructPropertyStorePathForControllerJob() {
```

Comment:
Do we also change the ZK location for existing controller jobs? I'd suggest making it a separate PR because that is not in the scope of this PR

Reply:
@saurabhd336 and I chatted about this and included it as part of this PR since it was touching related code. For instance, I also had to add rebalance jobType to the /table/tableName/{jobs} api, to retrieve all rebalance jobs. Given the small change specifically for reload, my request is to include this as part of this PR unless you strongly feel otherwise.

Reply:
@Jackie-Jiang and I discussed. Given this will be a backwards incompatible change, we will do this as a separate PR so that its easy to track which PR made that change. I'll keep reload_all and ZK_location the same for reload, force_commit, reload_all.
cc @saurabhd336